sdk-core/src/test/java/care/data4life/sdk/RecordServiceAdditionalResourceTypeModule.kt

```diff
@@ -942,8 +950,8 @@ class RecordServiceAdditionalResourceTypeModule {
                             SdkAttachmentFactory.wrap(firstAttachment),
                             SdkAttachmentFactory.wrap(secondAttachment)
                     ),
-                    any(),
-                    any()
+                    attachmentKey,
+                    USER_ID
             )
```

Comment:
Why is it important to remove any() ?

Reply:
`any()` in my opinion is never a good choice since you accept all payloads, since in a test environment the parameter should be known (except in some edge cases).  If you define the parameter specifically, the tests will not run and fail, which leads to much less problems with the test themselves.
Here I simply overlooked it the first time.